### DISCUSSION TITLE
Proyecto -Selección Dataset

### DISCUSSION THREAD

**Discussion by angiecoronel12** [rramosp/ai4eng.v1#158] [UDEA Students]
Hola!. Quisiéramos saber si el dataset que escogimos cumple con la condición del 10% de las variables categóricas, dado que algunas de estas se asignan como un valor numérico en el dataset (ejemplo: días de la semana - se asignan del 1 al 7), esta columna cumpliría como variable categórica?. Gracias! 

Link de acceso al dataset elegido: 
https://www.kaggle.com/datasets/benoit72/uk-accidents-10-years-history-with-many-variables?select=Accidents0514.csv

**Comment by rramosp** [comment]
hola angie ... sí, esa asignación del 1 al 7 la podemos considerar como
categórica

On Sat, Jul 2, 2022 at 8:44 PM angiecoronel12 ***@***.***>
wrote:

> Hola!. Quisiéramos saber si el dataset que escogimos cumple con la
> condición del 10% de las variables categóricas, dado que algunas de estas
> se asignan como un valor numérico en el dataset (ejemplo: días de la semana
> - se asignan del 1 al 7), esta columna cumpliría como variable categórica?.
> Gracias!
>
> Link de acceso al dataset elegido:
>
> https://www.kaggle.com/datasets/benoit72/uk-accidents-10-years-history-with-many-variables?select=Accidents0514.csv
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/158>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIPDPBYZMUGLECVBZ53VSDWA3ANCNFSM52P4ZYSQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Reply by angiecoronel12** [reply]
Profesor una pregunta, podría por favor recomendarnos alguna métrica de machine learning para una variable discreta, que no es clasificación binaria?.  En este caso la variable de salida seria 1,  2 o 3, que se refiere a la severidad de un evento. Gracias

**Comment by rramosp** [comment]
tienes varias opciones:

- puedes reportar cualquier métrica (accuracy, precision, recall, etc.) por
cada clase. Es como si hicieras tres clasificaciones binarias y midieras el
rendimiento de cada una (1 vs 2+3, 2 vs 1+3, 3 vs 1+2)
- tu variables es discreta pero *ordenada*, con lo que podrías también usar
alguna medida de regresión (RMSE, p.ej).

On Tue, Jul 5, 2022 at 11:18 AM angiecoronel12 ***@***.***>
wrote:

> Profesor una pregunta, podría por favor recomendarnos alguna métrica de
> machine learning para una variable discreta, que no es clasificación
> binaria?. En este caso la variable de salida seria 1, 2 o 3, que se refiere
> a la severidad de un evento. Gracias
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/158#discussioncomment-3086395>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIMWAUSKQ6WFXDAQFIDVSRN6FANCNFSM52P4ZYSQ>
> .
> You are receiving this because you commented.Message ID: <rramosp/ai4eng.
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
